client/lib/domains/utils/get-root-domain.ts

```diff
@@ -0,0 +1,14 @@
+import { getTld } from 'calypso/lib/domains';
+
+export function getRootDomain( domainName: string ) {
```

Comment:
```suggestion
export function getRootDomain( domainName: string ): string {
```
Just to be more explicit 🙂